.gitlab-ci.yml

```diff
@@ -58,3 +65,52 @@ test_acceptance:
       - report_*.html
     reports:
       junit: results_*.xml
+
+convert_raspbian:
+  stage: convert
+  image: teracy/ubuntu:18.04-dind-18.09.9
+  services:
+    - docker:18-dind
+  tags:
+    - mender-qa-slave
+  dependencies:
+    - build
+  before_script:
+    - apt update && apt install -yy bash wget unzip
+
+    - export IMAGE_NAME=$DOCKER_REPOSITORY:pr
+    - docker load -i image.tar
+
+    - wget -q ${RASPBIAN_URL}
+    - unzip ${RASPBIAN_NAME}.zip
+
+  script:
+    - echo "MENDER_CLIENT_VERSION=${MENDER_CLIENT_VERSION}" > mender_client_version_config
+    - env MENDER_ARTIFACT_NAME=mender-raspbian
```

Comment:
I think the Artifact name should include somehow the Raspbian version, maybe `mender-2019-09-26-raspbian-buster-lite`

Reply:
And possibly the mender version also.

Reply:
Right, I will change that for clarity. But note that it is anyway renamed in the publication stage to the final name.